sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -471,6 +470,7 @@ class QueueProperties(DictMixin):
     """The approximate number of messages contained in the queue."""
 
     def __init__(self, **kwargs: Any) -> None:
+        self.name = None #type: ignore [assignment]
```

Comment:
As discussed offline, we will type-ignore this for now and look to modify and correct the behavior further as a separate item: https://github.com/Azure/azure-sdk-for-python/issues/33389